drivers/feetech/feetech.c

```diff
@@ -29,12 +29,14 @@
 
 #define FEETECH_MAX_BUFFER_SIZE 128
 
-static size_t _feetech_stream_write(feetech_stream_t *stream, const uint8_t *buffer, size_t size) {
+static size_t _feetech_stream_write(feetech_stream_t *stream, const uint8_t *buffer, size_t size)
+{
     uart_write(stream->uart, buffer, size);
     return size;
 }
 
-static size_t _feetech_stream_read(feetech_stream_t *stream, uint8_t *buffer, size_t size) {
+static size_t _feetech_stream_read(feetech_stream_t *stream, uint8_t *buffer, size_t size)
+{
     uint32_t begin = xtimer_now_usec();
     uint32_t cur = begin;
     while(cur-begin < FEETECH_TIMEOUT_MS*1000) {
```

Comment:
Prefer putting space around arithmetic operators. It seems to be the convention used in other places in the code